src/main/java/com/kainos/ea/backend/services/CapabilityService.java

```diff
@@ -22,10 +23,15 @@ public CapabilityService(CapabilityRepository capabilityRepository) {
         return capabilityRepository.findAll();
     }
 
-    public Capability addCapability(Capability capability) throws InvalidNameException {
-        if (capability.getName().matches("[A-Za-z0-9 ]+")) {
-            return capabilityRepository.save(capability);
+    public Capability addCapability(Capability capability) throws InvalidNameException, InstanceAlreadyExistsException {
+        if (!capability.getName().matches("[A-Za-z0-9 ]+")) {
```

Comment:
Not raising as an issue but as a tip for the future, a line like this can take quite a while to read and understand, especially if there is Regex involved. I would suggest refactoring this line out into a well named function so that one doesn't have to wonder, "Hmmm I wonder what that Regex does". I know in this example they could look inside the function and read the error message but sometimes this might not be the case. 

Something like 
`if (!capability.getName().matches(ALPHANUMERIC_CHARS)) {` 
or 
`if (isAlphanumeric(capability.getName())) {`